test/unit/specs/switch.spec.js

```diff
@@ -127,6 +127,29 @@ describe('Switch', () => {
     });
   });
 
+  it('loading should show and switch should not respond to user click when in loading status', done => {
+    vm = createVue({
+      template: `
+        <div>
+          <el-switch loading v-model="value"></el-switch>
+        </div>
+      `,
+
+      data() {
+        return {
+          value: false
+        };
+      }
+    }, true);
+
+    vm.$el.querySelector('.el-switch__core').click();
+    expect(vm.$el.querySelector('.el-switch__core .circular')).to.exist;
+    Vue.nextTick(() => {
```

Comment:
We usually use `vm.$nextTick` here.

Reply:
I see, but i refer to the file `switch.spec.js`, maybe we need optimization here

https://github.com/ElemeFE/element/blob/dev/test/unit/specs/switch.spec.js#L124-L127
```
Vue.nextTick(() => {
    expect(vm.value).to.true;
    done();
});
```

Reply:
Interesting, I think we will normalize `$nextTick` usage across test codes in next major release.